resources/dicts/events/misc/mountainous.json

```
@@ -48,5 +48,17 @@
       "status": ["medicine cat", "elder"],
       "skill": ["LORE,1"]
     }
+  },
+  {
+    "event_id": "mtn_misc_any_patch_accessory",
+    "location": ["any"],
+    "season": ["any"],
+    "sub_type": ["accessory"],
+    "weight": 5,
+    "event_text":  "r_c found a little patch of acc_plural and decided to gift them to m_c",
+    "new_accessory": ["POPPY", "ORANGE POPPY", "CYAN POPPY", "WHITE POPPY", "PINK POPPY", "BULB WHITE"],
+    "r_c": {
+        "age": ["adolescent", "young adult", "adult", "senior adult", "senior"]
```

```suggestion
        "age": ["any"]
```
I think this is appropriate for any age.  It could also be nice to increase platonic like between r_c and m_c